### PR TITLE
[machine] Polishes instantiation of VM and removes if-wrapping around sendResponse() 

### PR DESCRIPTION
**Pull request by joelalejandro**
### Description

Adds some minor improvements to the VM code. See commits for more info.

### CODE REVIEW

File: packages/machine/src/vm.ts [hunk 1]
```diff
@@ -173,10 +176,12 @@ export class CounterfactualVM implements Observable {
...
     if (!execution.action.isAckSide) {
-      this.responseHandler.sendResponse(
-        new cf.node.Response(execution.action.requestId, status)
-      );
+      return;
     }
+
+    this.responseHandler.sendResponse(
+      new cf.node.Response(execution.action.requestId, status)
+    );
```
Comment (snario) [inline packages/machine/src/vm.ts]: Why the change?

Comment (joelalejandro) [reply packages/machine/src/vm.ts]: It's usually not good practice to have a single if-condition wrapping the entirety of a function. If for any reason, a function must abort its execution, it's preferred to do so as quickly as possible.

Comment (snario) [reply packages/machine/src/vm.ts]: Shouldn't the if clause be reversed though?

Comment (joelalejandro) [reply packages/machine/src/vm.ts]: Yeah, you're right on that, forgot to remove the `!`.
